.github/workflows/multichain-e2e.yml

```diff
@@ -0,0 +1,64 @@
+name: Multichain E2E Tests
+
+on:
+  # for now, only run on-demand
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  multichain-e2e:
+    runs-on: ubuntu-latest-16core
+    strategy:
+      matrix:
+        node-version: ['18.x']
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+          path: ./agoric-sdk
+      - uses: ./agoric-sdk/.github/actions/restore-node
+        with:
+          node-version: ${{ matrix.node-version }}
+          path: ./agoric-sdk
+
+      - name: Enable Corepack
+        run: corepack enable || sudo corepack enable
```

Comment:
pretty sure we should never need `sudo` in GH actions.

Reply:
Had source from [here](https://github.com/Agoric/agoric-3-proposals/blob/ca724af5f7090e7b75a670ef3fd25fa6ab2fdb8a/.github/workflows/ci.yml#L52) but since removed